src/Components/test/testassets/ComponentsApp.Server/wwwroot/index.html

```diff
@@ -9,8 +9,46 @@
     <link href="css/site.css" rel="stylesheet" />
 </head>
 <body>
+
+    <button class="btn btn-warning" onclick="window.Blazor.closeConnection()">Simulate connection close</button>
     <app>Loading...</app>
 
+    <div id="reconnect" class="modal" tabindex="-1" role="dialog" data-backdrop="static">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">Reconnect</h5>
+                </div>
+                <div class="modal-body">
+                    <p>Server connection lost.</p>
+                    <p class="text-warning"></p>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-primary">Reconnect</button>
+                </div>
+            </div>
+        </div>
+    </div>
+
     <script src="_framework/components.server.js"></script>
+
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>
```

Comment:
I know this is only test code, but it would be great not to have dependencies on jQuery and especially not CDNs. We definitely don't want to give the impression that people have to use jQuery with Components/Blazor :)

To avoid this, could we have a much simpler test UI that doesn't use jQuery or any fancy CSS?